app/javascript/components/rooms/Presentation.jsx

```diff
@@ -27,7 +27,7 @@ export default function Presentation() {
             </Card.Text>
             {/* TODO: Hadi - Change what is accepted as a file */}
             <input type="file" onChange={(e) => setPresentation(e.target.files[0])} accept="*" />
-            <Button variant="primary" className="w-100 my-3 py-2" onClick={() => handleUploadPresentation(presentation)}>
+            <Button variant="primary" className="w-100 my-3 py-2" onClick={() => onSubmit(presentation)}>
```

Comment:
Changed it back for now because I realised it was broken.. This is all going to change anyways when I go back to front-end